src/w3d/renderer/texture.cpp

```diff
@@ -215,7 +215,6 @@ TextureClass::TextureClass(unsigned width,
 #endif
 
     if (pool == POOL_DEFAULT) {
-        captainslog_assert(m_pool == POOL_DEFAULT);
```

Comment:
This assert is obsolete. It's always true with no chance being false ever.

Reply:
it checks m_pool not pool